### ISSUE TITLE
Mislabelled quarters in example

### ISSUE THREAD

**Issue by d-giles** [bmorris3/interp-acf#5]
The full quarters of data available for KOI 3.01, and retrieved in the example, are quarters 1-6, 8-10,12-14, 16, and 17 rather than Q0 to 13.

**Comment by bmorris3** [comment]
Good point. Do you get the _same_ rotation period when you take this into account?

**Comment by d-giles** [comment]
Indeed they do, Stela and I used Lightkurve to get the SAP fluxes and got the exact same rotation periods. We also used the kplr API to download the data and checked the start times for each quarter, they matched them to the quarters from Lightkurve.
